### PR TITLE
fix(public-docsite): Add reset for svgs and update cdn to latest release

### LINKED ISSUE #25868: Updates to Fabric Core [Epic]
With an update adding the new m365 icons, this is a good time to do a full update on Fabric Core

- [x] Add new m365 icons to the Core icon set (leave the old office ones in there as deprecated)
- [x] Update **all** CDN references to use new cdn url (https://res-1.cdn.office.net/files/fabric-cdn-prod_20221201.001/assets/)
- [ ] update v8 website to use new version of core (and anywhere else using office.com logo) to show new m365 icons instead

### PR DESCRIPTION
**Pull request by sopranopillow**
### SVG fix
image

### SVG fix
image

### CDN

Our docs now reference and use `11.1.0` along with the updated cdn link.

## Related Issue(s)

* Fixes #25868

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/983c8a8477ffb675d5c756b5407d926bb90a5ad7'>983c8a8477ffb675d5c756b5407d926bb90a5ad7</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/pr-deploy-site/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <title>PR Deployed Sites</title>
     <link
       rel="stylesheet"
-      href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css"
+      href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.1.0/css/fabric.min.css"
```
Comment (micahgodbolt) [inline apps/pr-deploy-site/index.html:8]: we want to use the new res-1.cdn everywhere, including here

**Comment by fabricteam** [comment]
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 4 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26005&commitId=8099fdcf9f6bdc8f41db5dd7ba1a1c0a39d647d7&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1041 screenshots unchanged, 0 screenshots with different dimensions and 4 screenshots with visible difference.


  <details><summary><b>unknown</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DocumentCard.Compact with preview image.default.chromium.png|5| Changed|
|DocumentCard.Compact with preview list.default.chromium.png|3| Changed|
|DocumentCard.Not truncated.default.chromium.png|4| Changed|
|DocumentCard.With secondary title style.default.chromium.png|4| Changed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/BaseButton-mount_base.html">1185</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/BaseButton-mount.html">1160</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Breadcrumb-mount_base.html">2962</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Breadcrumb-mount.html">2987</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Checkbox-mount_base.html">2602</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Checkbox-mount.html">2633</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/CheckboxBase-mount_base.html">2336</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/CheckboxBase-mount.html">2330</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ChoiceGroup-mount_base.html">4373</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ChoiceGroup-mount.html">4451</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ComboBox-mount_base.html">1238</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ComboBox-mount.html">1231</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/CommandBar-mount_base.html">9482</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/CommandBar-mount.html">9575</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ContextualMenu-mount_base.html">10918</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ContextualMenu-mount.html">10799</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/DefaultButton-mount_base.html">1348</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/DefaultButton-mount.html">1379</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/DetailsRow-mount_base.html">3607</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/DetailsRow-mount.html">3577</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/DetailsRowFast-mount_base.html">3586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/DetailsRowFast-mount.html">3595</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/DetailsRowNoStyles-mount_base.html">3416</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/DetailsRowNoStyles-mount.html">3453</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Dialog-mount_base.html">3045</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Dialog-mount.html">3026</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/DocumentCardTitle-mount_base.html">571</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/DocumentCardTitle-mount.html">578</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Dropdown-mount_base.html">3171</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Dropdown-mount.html">3177</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/FocusTrapZone-mount_base.html">1970</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/FocusTrapZone-mount.html">1969</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/FocusZone-mount_base.html">1955</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/FocusZone-mount.html">1964</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedList-mount_base.html">1884</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedList-mount.html">2138</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedList-virtual-rerender_base.html">1113</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedList-virtual-rerender.html">1136</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1662</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedList-virtual-rerender-with-unmount.html">1652</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedListV2-mount_base.html">572</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedListV2-mount.html">565</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedListV2-virtual-rerender_base.html">548</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedListV2-virtual-rerender.html">547</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">559</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">567</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/IconButton-mount_base.html">1886</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/IconButton-mount.html">1896</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Label-mount_base.html">719</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Label-mount.html">728</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Layer-mount_base.html">4342</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Layer-mount.html">4325</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Link-mount_base.html">825</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Link-mount.html">825</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/MenuButton-mount_base.html">1658</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/MenuButton-mount.html">1652</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/MessageBar-mount_base.html">2348</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/MessageBar-mount.html">2372</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Nav-mount_base.html">3233</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Nav-mount.html">3235</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/OverflowSet-mount_base.html">1347</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/OverflowSet-mount.html">1360</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Panel-mount_base.html">2536</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Panel-mount.html">2545</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Persona-mount_base.html">1313</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Persona-mount.html">1311</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Pivot-mount_base.html">1626</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Pivot-mount.html">1645</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/PrimaryButton-mount_base.html">1504</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/PrimaryButton-mount.html">1499</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Rating-mount_base.html">6942</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Rating-mount.html">6906</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/SearchBox-mount_base.html">1513</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/SearchBox-mount.html">1486</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Shimmer-mount_base.html">2868</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Shimmer-mount.html">2830</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Slider-mount_base.html">2081</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Slider-mount.html">2071</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/SpinButton-mount_base.html">4620</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/SpinButton-mount.html">4626</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Spinner-mount_base.html">794</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Spinner-mount.html">810</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/SplitButton-mount_base.html">3040</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/SplitButton-mount.html">3052</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Stack-mount_base.html">823</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Stack-mount.html">823</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/StackWithIntrinsicChildren-mount_base.html">2323</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/StackWithIntrinsicChildren-mount.html">2375</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/StackWithTextChildren-mount_base.html">4823</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/StackWithTextChildren-mount.html">4800</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/SwatchColorPicker-mount_base.html">10433</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/SwatchColorPicker-mount.html">10504</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/TagPicker-mount_base.html">2624</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/TagPicker-mount.html">2629</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/TeachingBubble-mount_base.html">84452</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/TeachingBubble-mount.html">84484</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Text-mount_base.html">784</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Text-mount.html">785</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/TextField-mount_base.html">1566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/TextField-mount.html">1570</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ThemeProvider-mount_base.html">1496</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ThemeProvider-mount.html">1513</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ThemeProvider-virtual-rerender_base.html">1067</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ThemeProvider-virtual-rerender.html">1070</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2139</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2148</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Toggle-mount_base.html">1093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/Toggle-mount.html">1086</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/buttonNative-mount_base.html">542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26005/perf-test/buttonNative-mount.html">541</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

File: apps/public-docsite/src/styles/_base.scss [hunk 2]
```diff
@@ -35,6 +35,11 @@
     border: none;
   }
 
+  // SVGs
+  svg {
```
[the remark this answers is not in the document]
Comment (sopranopillow) [reply apps/public-docsite/src/styles/_base.scss]: How's this? I checked that those icons are the only ones with that className

```
  // SVGs
  .ms-HomePage-cardIcon svg {
    vertical-align: unset;
  }
```

[the remark this answers is not in the document]
Comment (sopranopillow) [reply apps/public-docsite/src/styles/_base.scss]: Also, I checked and those styles are after

**Review by micahgodbolt — APPROVED** [review]

**Review by khmakoto — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]
Approving for the changes to the cdn for react-northstar docsite

**Pull request merged**